lua/rzls/init.lua

```diff
@@ -37,7 +37,9 @@ local defaultConfg = {
 
 ---@type lsp.ClientCapabilities
 local extraCapabilities = {
-    colorProvider = true,
+    textDocument = {
+        colorProvider = {},
+    },
```

Comment:
```suggestion
    },
```

I dont think we need to set this at all, and we let the plugins configure it